permissions/plugin-cloudbees-feature-management.yml

```diff
@@ -0,0 +1,13 @@
+---
+name: "cloudbees-feature-management"
+github: &GH "jenkinsci/cloudbees-feature-management-plugin"
+cd:
+  enabled: true
+paths:
+- "io/jenkins/plugins/cloudbees-feature-management"
+developers:
+- "tfennelly"
+- "agentgonzo"
+- "pawoolley"
```

Comment:
> User name not known to Artifactory and Jira: pawoolley

It looks like @pawoolley doesn't have a login in artifactory. We can proceed without him for now and add him later as required.
```suggestion
```

Reply:
@timja @dduportal - do you have permissions to change this PR with ☝️  and see if it succeeds the build this time?